docker-compose.yml

```diff
@@ -25,13 +25,13 @@ x-env-defaults: &env
   YARN_CACHE_FOLDER: /.yarn-cache
 
 x-env-aerilon: &env-aerilon-staging
-  GRAPHQL_URI: ${GRAPHQL_URI-https://aerilon.graphql.base-cms-staging.io}
+  GRAPHQL_URI: ${GRAPHQL_URI-https://aerilon.graphql.base-cms.io}
   OEMBED_URI: ${OEMBED_URI-https://aerilon.oembed.base-cms-staging.io}
   RSS_URI: ${RSS_URI-https://aerilon.rss.base-cms-staging.io}
   SITEMAPS_URI: ${SITEMAPS_URI-https://aerilon.sitemaps.base-cms-staging.io}
 
 x-env-picon: &env-picon-staging
-  GRAPHQL_URI: ${GRAPHQL_URI-https://picon.graphql.base-cms-staging.io}
+  GRAPHQL_URI: ${GRAPHQL_URI-https://picon.graphql.base-cms.io}
   OEMBED_URI: ${OEMBED_URI-https://picon.oembed.base-cms-staging.io}
   RSS_URI: ${RSS_URI-https://picon.rss.base-cms-staging.io}
   SITEMAPS_URI: ${SITEMAPS_URI-https://picon.sitemaps.base-cms-staging.io}
```

Comment:
Was this intentional?  If so, we might want to remove the entire staging blocks (Lines 27-37) and push that in a separate PR

Reply:
It is only graphql staging which doesn't exist anymore. You can't run a local docker setup without removing the staging part.

Reply:
Makes sense, but while we're at it, we might as well remove the rest of them since all staging was removed

Reply:
Ah, I didn't realize all staging was removed. I thought it was just graphql.

Reply:
@CaitlynKnudsvig - I've removed as much as I can without breaking the build.

Reply:
No no, I meant remove the entire lines of code, not just `-staging`, though I haven't tested it so I can't say for sure.  @Jlaird correct me if I'm wrong on this

Reply:
Yeah, I tried removing the whole thing, but that broke it because the brands are expecting a reference to their specific one (picon or aerilon), and the environment setting below those sections is expecting the URI information.

Reply:
Well shit, then idk 🤷 I guess if removing just `-staging` works and you're still able to spin up a couple sites then we can just go with it.  I've run out of fucks.